noxfile.py

```diff
@@ -200,6 +200,18 @@ def lint(session: nox.Session) -> None:
     mypy(session)
 
 
+@nox.session(python="3.11")
+def pyodineconsole(session: nox.Session) -> None:
+    # build wheel into dist folder
+    session.install("build")
+    session.run("python", "-m", "build")
+    session.run(
+        "cp", "templates/pyodide-console.html", "dist/index.html", external=True
+    )
+    session.cd("dist")
+    session.run("python", "-m", "http.server")
```

Comment:
Let's specify a port here and then open the browser automatically, I believe there's a Python function for that?

Reply:
but I don't think we can let the `python -m http.server` in the background with nox. So we can open the browser before the server is up , but that would be great either.

Reply:
Oh right, yeah let's keep this as-is!

Reply:
I can create a python script to create virtualenv , install dependencies, etc instead. then we are not bound by the nox rules.